### DISCUSSION TITLE
Anti-features in 1.6.1

### DISCUSSION THREAD

**Discussion by serrq** [aritra-tech/Notify#204] [General]
![1000094422](https://github.com/aritra-tech/Notify/assets/113377431/c7e2aff0-9a9a-40bb-8f59-95583bc820db)

**Comment by aritra-tech** [comment]
It's coming because of `In-App Update` feature.

**Reply by serrq** [reply]
The fdroid version has the `in-app update` too? I can understand for the Github one...

**Reply by aritra-tech** [reply]
The In-App update is for the users who downloaded through play store

**Reply by serrq** [reply]
There is no reason to give internet permission to a fdroid app.
By the way app is your and you do what you want.
